pylasu/validation/validation.py

```diff
@@ -39,3 +39,8 @@ def semantic(message: str, severity: IssueSeverity = IssueSeverity.ERROR, positi
 class Result:
     root: Node
     issues: List[Issue] = field(default_factory=list)
```

Comment:
Should this extend WithIssues?

Reply:
You are right, I am just not sure of how the field definition works with inheritance, but I made the change